README.md

```diff
@@ -305,11 +317,17 @@ ruff format .
 If you would prefer to see what would change you can:
 
 ```shell
-ruff format --check .
+ruff format --check . # just tells you that things would change, e.g. "1 file would be reformatted, 4 files already formatted"
+ruff format --diff .  # if files would be reformatted: print the diff between the current file and the re-formatted file, then exit with non-zero status code
```

Comment:
i sort of like being able to see the diff before i apply it (though i know we'll almost always just take the suggested formatting, so maybe this isn't worth the extra line of text 🤷).